src/bootstrap.js

```diff
@@ -2,9 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-/* globals XPCOMUtils, UAOverrider UAOverrides */
+/* globals XPCOMUtils, Services, UAOverrider UAOverrides */
 
 Components.utils.import("resource://gre/modules/XPCOMUtils.jsm");
+
+const UA_ENABLE_PREF_NAME = "extensions.webcompat.perform_ua_overrides";
```

Comment:
(drive-by comment) 🚗 

So currently we only do UA overrides, so this seems fine. But in the future, do we envision a single pref to enable everything that this add-on can do? If so, does it make sense to make a more general pref?

(also, is the extensions namespace (prefix??) the right place for this? no idea!)